tests/integration/service/test_external_fs_service.py

```diff
@@ -0,0 +1,122 @@
+"""
+Integration tests for ExternalDatabricksFeatureStoreService.
+"""
+import os
+import time
+from datetime import datetime
+
+import pytest
+from databricks import sql as databricks_sql
+
+from featurebyte.service.external_feature_store import ExternalDatabricksFeatureStoreService
+
+
+@pytest.fixture(name="external_fs_service")
+def external_fs_service_fixture():
+    """
+    Fixture for ExternalFsService
+    """
+    return ExternalDatabricksFeatureStoreService()
+
+
+@pytest.fixture(name="databricks_connection")
```

Comment:
maybe call this `databricks_unity_connection` to differentiate from the non-unity connection